docs/configuration.md

```diff
@@ -76,6 +76,86 @@ Defaults to enabled.
 
 ---
 
+### credential.trace
+
+Enables trace logging of all activities.
+Configuring Git and GCM to trace to the same location is often desirable, and
+GCM is compatible and cooperative with `GIT_TRACE`.
+
+#### Example
+
+```shell
+git config --global credential.trace /tmp/git.log
+```
+
+If the value of `credential.trace` is a full path to a file in an existing
+directory, logs are appended to the file.
+
+If the value of `credential.trace` is `true` or `1`, logs are written to
+standard error.
+
+Defaults to tracing disabled.
+
+**Also see: [GCM_TRACE][gcm-trace]**
```

Comment:
Links are resulting in 404s. Will be resolved once merged?

Reply:
Hmm.. they are correct.. so I assume some issue with the PR view.